zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotChunkReader.java

```diff
@@ -91,7 +89,7 @@ public ByteBuffer nextId() {
       return null;
     }
 
-    return encodeChunkId(chunksView.first().toString() + "__" + offset);
+    return new SnapshotChunkId(chunksView.first().toString() + "__" + offset).getId();
```

Comment:
:wrench: We might as well make this part of `SnapshotChunkId`. Like a constructor or static method taking a filename and offset. Same for the splitting in line 79 then.